src/app/Http/Controllers/Api/MovieController.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Http\Controllers\Controller;
+use App\Models\Movie\Movie;
+use Illuminate\Http\Request;
+
+class MovieController extends Controller
+{
+    public function add(Request $request)
+    {
+        $data = $request->validate([
+            'title' => 'required|max:255',
+            'description' => 'required',
+            'coverImage' => 'required',
+            'genre' => 'required'
+        ]);
+        Movie::create([
+            'title' => $data['title'],
+            'description' => $data['description'],
+            'coverImage' => $data['coverImage'],
+            'genre' => $data['genre'],
+        ]);
+        return response('success', 200);
+    }
+
+    public function getall()
+    {
+        return response(Movie::all(), 200);
+    }
+}
```

Comment:
https://laravel.com/docs/9.x/controllers#api-resource-routes

unutar laravela odradi `php artisan make:controller --model=Movie` i videces da ce sam da ti generise neke metode. Svaka ta metoda sluzi necemu. Istrazi to i koristi te metode